redesign/src/404.html

```diff
@@ -0,0 +1,41 @@
+---
+layout: layouts/default.njk
+title: 404 | Expunge Maryland 
+---
+
+{% include "_nav.html" %}
+{% include "_header.html" %}
+
+<section class= "site-wrapper">
+
+    <section class="main error-page">
+        <button class= "back u-button-reset" onclick="goBack()">
+            <p class="back--text"> Back </p>
+                <section class="back--icon">
+                <div class="back--arrow"></div> <hr>
+                </section>
+        </button>
+         
+        <section class="outline-title"> <p class="error--title">404</p> </section>
+        <section class="error border u-center"> 
+        <h2 class="error--text">Unfortunately we can’t be of further help to you with this information. </h2>
+            <section clas="options u-center">
+                 <div class="options--flex">
+                    <p class="error--option">If you don’t know because you don’t have your official criminal record, you can; see the steps to optaining your arrest information.</p>
```

Comment:
```suggestion
                    <p class="error--option">If you don’t know because you don’t have your official criminal record, you can see the steps to obtaining your arrest information.</p>
```
Two little typos here! 👁